platform/darwin/docs/guides/Predicates and Expressions.md

```diff
@@ -53,9 +53,17 @@ The following aggregate operators are supported:
 `NSInPredicateOperatorType`       | `key IN { 'iOS', 'macOS', 'tvOS', 'watchOS' }`
 `NSContainsPredicateOperatorType` | `{ 'iOS', 'macOS', 'tvOS', 'watchOS' } CONTAINS key`
 
-Operator modifiers such as `c` (for case insensitivity), `d` (for diacritic
-insensitivity), and `l` (for locale sensitivity) are unsupported for comparison
-and aggregate operators that are used in the predicate.
+The following comparison predicate options are supported for comparison and
+aggregate operators that are used in the predicate:
+
+`NSComparisonPredicateOptions`          | Format string syntax
+----------------------------------------|---------------------
+`NSCaseInsensitivePredicateOption`      | `'QUEBEC' =[c] 'Quebec'`
+`NSDiacriticInsensitivePredicateOption` | `'Québec' =[d] 'Quebec'`
+
+Other comparison predicate options are unsupported, namely `l`
```

Comment:
To make sure I'm understanding: if you wanted to explicitly do locale-sensitive comparison with something other than the current locale, you'd have to manually craft your expression, because NSPredicate only supports a binary "locale-sensitive"?

Reply:
I made some last-minute changes so that this predicate would work:

* The collator options object is no longer wrapped in a `literal` expression when converting from NSExpression to JSON.
* The `MGL_FUNCTION()` syntax is used instead of the `==[…]` syntax when a `locale` is specified.